src/swapnilMaheshwari/Assignment21/SecondLargestNumInArray.java

```diff
@@ -0,0 +1,34 @@
+/*
+ Return second highest from given array.
+input : {10,99,33,44,89,76}
+output : 89 
+ 
+ */
+
+package swapnilMaheshwari.Assignment21;
+
+public class SecondLargestNumInArray {
+
+	int getSecondLarNum(int[] arr) {
+		int maxNum1 = 0, maxNum2 = 0;
+
+		for (int i = 0; i < arr.length; i++) {
+			if (maxNum1 < arr[i])
+				maxNum1 = arr[i];
+		}
+
+		for (int j = -0; j < arr.length; j++) {
```

Comment:
why do we need two loops, you can handle this in a single loop as well.

with else if(..) condition